playbooks/ae/templates/udp_data_mapping.json.j2

```diff
@@ -0,0 +1,264 @@
+{
```

Comment:
You may want to rename this file udp_data_mapping.json as this isn't really a template.